components/apps_form/apps_form_field/apps_form_select_field.tsx

```diff
@@ -127,6 +128,8 @@ export default class AppsFormSelectField extends React.PureComponent<Props, Stat
             selectComponent = this.renderDynamicSelect();
```

Comment:
Both are using renderDynamicSelect, don't know if it's intended, but in that case you can do:

```typescript
if (['dynamic_select', AppfieldTypes.USER].includes(field.type))
  ...
  this.renderDynamicSelect()
```

Reply:
actually, it was intended for now as both Dynamic and User Select renderers had completely same code, will separate them when needed while implementing custom renderers.